packages/web-react/config/rollup.config.js

```diff
@@ -1,6 +1,7 @@
+import path from 'path';
 import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
-import path from 'path';
+// eslint-disable-next-line import/order
```

Comment:
// eslint-disable-next-line import/order -- conflicting no empty line in import groups with an empty line after import statement (require is also "import")
import { terser as minify } from 'rollup-plugin-terser';